make_var_mx6ul_dart_debian.sh

```diff
@@ -768,15 +768,43 @@ function make_kernel_modules() {
 	return 0;
 }
 
+###################### Twonav kernel package fix for older kernel  packages ##########
+function pkg_info_gen() {
+	local OLD_KERN_BRANDS="twonav os"
+	local OLD_KERN_MODELS="trail aventura"
+	local BREAK_VERSION="1.0.19"
+	echo -n "$1"
+	local i j cnt=0
+	for i in $OLD_KERN_BRANDS ; do for j in $OLD_KERN_MODELS ; do
+		if ((cnt++ > 0)) ; then echo "," ; fi
+		echo -en "\t$2-4.1.15-$i-$j-2018 (<= $BREAK_VERSION)"
```

Comment:
@lluisdiaz may be the 4.1.15 and the 2018 may be unhardwired (?), also the BREAK_VERSION could be moved as a global variable (?)

Reply:
- why to move BREAK_VERSION to a global? this is some kind of patch and this way is localized.
- this debian script is based on 4.1.15 kernel version and our way to handle this devices is with 2018, i see no porblem with that.. or I don't understand the question :)